ldb/templates/ldb/person_detail.html

```diff
@@ -87,9 +87,8 @@ <h2 class="page-header">
           {% stcontrol_format person 'email' '<a href="mailto:{0}">{0}</a>' %}
           {% stcontrol_format person 'phone_fixed' '<a href="tel:{0}"><i class="fa fa-fw fa-phone"></i>{0}</a>' %}
           {% stcontrol_format person 'phone_mobile' '<a href="tel:{0}"><i class="fa fa-fw fa-mobile"></i>{0}</a>' %}
-          {% if person.student %}
-            {% stcontrol_format person.student 'phone_parents' '<a href="tel:{0}"><i class="fa fa-fw fa-phone"></i>{0}</a>' %}
-          {% endif %}
+          {% stcontrol person.student 'emergency_name' %}
```

Comment:
The `if person.student` is removed. Does that break something or was it unnecessary before?

Reply:
Good catch. This caused errors when no emergency_name was given. I've moved it back to inside an `if person.student` block.